.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for your interest in the project. Bugs filed and PRs submitted are appreciated!
+
+Please make sure that you are familiar with and follow the Code of Conduct for
+this project (found in the CODE_OF_CONDUCT.md file).
+
+Also, please make sure you're familiar with and follow the instructions in the
+contributing guidelines (found in the CONTRIBUTING.md file).
+
+If you're new to contributing to open source projects, you might find this free
+video course helpful: http://kcd.im/pull-request
+
+Please fill out the information below to expedite the review and (hopefully)
+merge of your pull request!
+-->
+
+<!-- What changes are being made? (What feature/bug is being fixed here?) -->
+**What**:
```

Comment:
Thoughts on adding an example on how to fill this out?
```
e.g. Fixes #0
```

Reply:
Yeah, that or `re #0` or other activation keywords (like `depends on`).

Reply:
Added example and a link to GitHub's list of valid keywords.

Reply:
Added the example and a link to GitHub's valid keyword list.

Reply:
Awesome!!